Keas.Mvc/Controllers/TagsManagerController.cs

```diff
@@ -38,11 +38,28 @@ public ActionResult Create()
         // POST: Tags/Create
         [HttpPost]
         public async Task<ActionResult> Create( Tag newTag)
-        {
+        {
+            var team = await _context.Teams.FirstAsync(t => t.Slug == Team);
+            if (ModelState.IsValid)
+            {
+                if (newTag.Name.Contains(","))
```

Comment:
Instead why don't we validate w/ a regex in case we want to add other stuff.  Then we can just do normal regex validation on the name, without having to check.  Of course our regex could just check for commas, but I think we might actually want to just only allow `\w` alpha numeric